scripts/check_rust_dependency.js

```diff
@@ -28,6 +28,10 @@ function getRepeatDeps() {
 }
 
 async function main() {
+	// 'bitflags': napi has upgraded to latest `bitfalgs@2.x.x`, but there are still lots of dependencies still use `bitflags@1.x.x`, like `clap, swc`,
+	// this cause CI failed in version checking, `bitflags@2.x.x` still need some time to adopt in rust community, but we need upgrade napi-rs to latest to fix some bug.
+	// so bypass `bitflags` for now.
+	let byPassDepList = ['bitflags']
```

Comment:
The `ignore_deps` on line 8 also has the same effect, leaving one is enough

Reply:
Done